### PR TITLE
Fix android time picker returning today's date instead of the given date

### LINKED ISSUE #94: Wrong value returned from picker
# Bug

Time picker returns today's date instead of given date.

In `datetimepicker.android.js` 53rd line should be `const date = new Date(value);` instead of `const date = new Date();`

### PR DESCRIPTION
**Pull request by roconnorr**
# Summary

After a time value is chosen in the android picker, use the value prop to create the date object so it doesn't default to returning the current date when using the time picker. Closes #94

### What's required for testing (prerequisites)?

A test project for android with a time picker set to ```mode="time"```

### What are the steps to reproduce (after prerequisites)?

Pass the time picker a date that is not today, log the value returned from the picker after selecting a time, and see that it returns today's date.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |

## Checklist

- [x] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md` - not applicable for bug fix
- [x] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow) - not applicable for bug fix
- [ ] I added a sample use of the API in the example project (`example/App.js`) - not applicable for bug fix

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+### 2.2.2
```
Comment (luancurti) [inline CHANGELOG.md]: Can you change to `2.2.1`?

Comment (roconnorr) [reply CHANGELOG.md]: Yup I have changed it, looks like there is already a tag in the repo for 2.2.1 though?

Comment (luancurti) [reply CHANGELOG.md]: You are right, I'll fix this in another PR thanks

**Comment by kptac** [comment]
This issue has not been fixed still. Using the latest v 2.3.0, On setting time, the date still changes to the current date.

**Comment by lander854** [comment]
same here

**Comment by kptac** [comment]
@roconnorr - kindly look into this issue again. The issue is still not resolved. Thanks

**Pull request merged**